New file for reviewable

asheshjain399 opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mbenisch/tensorflow/2)
<!-- Reviewable:end -->

Commit:
New file for reviewable

mbenisch commented:
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[tensorflow/core/kernels/matrix_set_diag_op_reviable.cc, line 44 at r1](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p50MEUe-LG9HRefcl:-L3p50MEUe-LG9HRefcm:b-gjmfex) ([raw file](https://github.com/mbenisch/tensorflow/blob/00afceb6bbfc305c472646c2bee91ad560ce4b1d/tensorflow/core/kernels/matrix_set_diag_op_reviable.cc#L44)):*
> ```cpp
> template <typename Device, typename T>
> class MatrixSetDiagOp : public OpKernel {
>  public:
> ```

kkjdkfjg

---

*[tensorflow/core/kernels/matrix_set_diag_op_reviable.cc, line 51 at r1](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p4wGWWKzdQnh7i6VR:-L3p4wGWWKzdQnh7i6VS:b-h8917w) ([raw file](https://github.com/mbenisch/tensorflow/blob/00afceb6bbfc305c472646c2bee91ad560ce4b1d/tensorflow/core/kernels/matrix_set_diag_op_reviable.cc#L51)):*
> ```cpp
>     const Tensor& diag = context->input(1);
> 
>     const TensorShape& input_shape = input.shape();
> ```

this is wrong

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mbenisch/tensorflow/2)*
<!-- Sent from Reviewable.io -->

Commit:
More changes

Commit:
More edits

Commit:
Delete code

mbenisch commented:
Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---

*[a discussion](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p6kntcxg1iMmNiT6v:-L3p6kntcxg1iMmNiT6w:b-d21fj3) (no related file):*
ksjdflksjkldfjs

---

*[tensorflow/core/kernels/listdiff_op.cc, line 71 at r2](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p6mV3AbD_NT55gmVf:-L3p6mV3AbD_NT55gmVg:b6zcrzn) ([raw file](https://github.com/mbenisch/tensorflow/blob/cae2587e0c99f7dcef7b9e80e71c8f39a42c26a4/tensorflow/core/kernels/listdiff_op.cc#L71)):*
> ```cpp
>     for (Tidx i = 0, p = 0; i < static_cast<Tidx>(x_size); ++i) {
>       if (y_set.count(Tx(i)) == 0) {
>         OP_REQUIRES(context, p < out_size,
> ```

jksjdfkls

---

*[tensorflow/core/kernels/matrix_band_part_op.cc, line 92 at r2](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p6ncCDZxFqkJFNNj2:-L3p6ncCDZxFqkJFNNj3:b2u1q0p) ([raw file](https://github.com/mbenisch/tensorflow/blob/0d79ba6ef0c70d05e7acf9e9587bf4a9e8dae31d/tensorflow/core/kernels/matrix_band_part_op.cc#L92)):*
> 
skjdflkjs

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mbenisch/tensorflow/2)*
<!-- Sent from Reviewable.io -->

asheshjain399 commented:
Review status: 0 of 3 files reviewed at latest revision, 5 unresolved discussions.

---

*[tensorflow/core/kernels/listdiff_op.cc, line 71 at r2](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p6mV3AbD_NT55gmVf:-L3p8BBU5LoyGPe3-wJC:bfbco06) ([raw file](https://github.com/mbenisch/tensorflow/blob/cae2587e0c99f7dcef7b9e80e71c8f39a42c26a4/tensorflow/core/kernels/listdiff_op.cc#L71)):*
<details><summary><i>Previously, mbenisch (Michael Benisch) wrote…</i></summary><blockquote>

jksjdfkls
</blockquote></details>

Hmm .. okay I addressed your comments. 

---

*[tensorflow/core/kernels/matrix_set_diag_op_reviable.cc, line 44 at r1](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p50MEUe-LG9HRefcl:-L3p8FnCJXe2Vo_ALJYj:bb7k0qh) ([raw file](https://github.com/mbenisch/tensorflow/blob/00afceb6bbfc305c472646c2bee91ad560ce4b1d/tensorflow/core/kernels/matrix_set_diag_op_reviable.cc#L44)):*
<details><summary><i>Previously, mbenisch (Michael Benisch) wrote…</i></summary><blockquote>

kkjdkfjg
</blockquote></details>

lgtm

---

*[tensorflow/core/kernels/matrix_set_diag_op_reviable.cc, line 51 at r1](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L3p4wGWWKzdQnh7i6VR:-L3p8GgOFLUDs4OXDNLi:b2nifei) ([raw file](https://github.com/mbenisch/tensorflow/blob/00afceb6bbfc305c472646c2bee91ad560ce4b1d/tensorflow/core/kernels/matrix_set_diag_op_reviable.cc#L51)):*
<details><summary><i>Previously, mbenisch (Michael Benisch) wrote…</i></summary><blockquote>

this is wrong
</blockquote></details>

wow

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mbenisch/tensorflow/2)*
<!-- Sent from Reviewable.io -->

asheshjain399 commented:
Reviewed 1 of 1 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[tensorflow/core/kernels/matrix_band_part_op.cc, line 89 at r2](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L4EBY7zRdzHo_PgySBn:-L4EBY7zRdzHo_PgySBo:bb7jfon) ([raw file](https://github.com/mbenisch/tensorflow/blob/0d79ba6ef0c70d05e7acf9e9587bf4a9e8dae31d/tensorflow/core/kernels/matrix_band_part_op.cc#L89)):*
> ```cpp
>       context->set_output(0, input);
>       return;
>     }
> ```

kjhk

---

*[tensorflow/core/kernels/matrix_set_diag_op_reviable.cc, line 42 at r1](https://reviewable.io:443/reviews/mbenisch/tensorflow/2#-L4EBZUl3Hnxf-O2jEEp:-L4EBZUm9c1MLcbjz7rE:bo89vt5) ([raw file](https://github.com/mbenisch/tensorflow/blob/00afceb6bbfc305c472646c2bee91ad560ce4b1d/tensorflow/core/kernels/matrix_set_diag_op_reviable.cc#L42)):*
> ```cpp
> typedef Eigen::GpuDevice GPUDevice;
> 
> template <typename Device, typename T>
> ```

kjlhkjh

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mbenisch/tensorflow/2)*
<!-- Sent from Reviewable.io -->
